src/Mailer/Email.php

```diff
@@ -1925,6 +1929,25 @@ protected function _renderTemplates($content)
             $rendered[$type] = rtrim($rendered[$type], "\n");
         }
 
+        /* Embed images inline in html templates */
+        if (!empty($rendered['html'])) {
+            $rendered['html'] = str_replace(['file:', 'file://', 'cid://'], 'cid:', $rendered['html']);
```

Comment:
I don't think blindly replacing every time we send an email is a great idea. Wouldn't it be better to tell the email class that it can expect the images there and where to look them for?